src/app/components/MediaLoader/index.styles.ts

```diff
@@ -1,20 +1,77 @@
+import pixelsToRem from '#app/utilities/pixelsToRem';
 import { css, Theme } from '@emotion/react';
 
+const commonMarginSpacing = ({ mq, spacings }: Theme) =>
+  css({
+    marginInline: `${spacings.FULL}rem`,
+    [mq.GROUP_2_MIN_WIDTH]: {
+      marginInline: `${spacings.DOUBLE}rem`,
+    },
+    [mq.GROUP_4_MIN_WIDTH]: {
+      marginInline: 0,
+    },
+  });
+
+export const mediaPortraitStyles = ({ mq }: Theme) => [
+  css({
+    aspectRatio: '9 / 16',
+    maxWidth: `${pixelsToRem(185)}rem`,
+    [mq.GROUP_1_MIN_WIDTH]: {
+      maxWidth: `${pixelsToRem(256)}rem`,
+    },
+    [mq.GROUP_2_MIN_WIDTH]: {
+      maxWidth: `${pixelsToRem(274)}rem`,
+    },
+    [mq.GROUP_3_MIN_WIDTH]: {
+      maxWidth: `${pixelsToRem(200)}rem`,
+    },
+    [mq.GROUP_3_MIN_WIDTH]: {
+      maxWidth: `${pixelsToRem(190)}rem`,
+    },
+  }),
+  commonMarginSpacing,
+];
+
 export default {
   figure: ({ spacings }: Theme) =>
     css({
       margin: 0,
       paddingBottom: `${spacings.TRIPLE}rem`,
       width: '100%',
     }),
+
   liveRadioMediaContainer: ({ palette }: Theme) =>
     css({
       backgroundColor: palette.WHITE,
       height: '165px',
     }),
-  mediaContainer: ({ palette }: Theme) =>
+
+  mediaContainerLandscape: ({ palette }: Theme) =>
     css({
       backgroundColor: palette.BLACK,
       aspectRatio: '16 / 9',
     }),
+
+  mediaContainerPortrait: ({ palette }: Theme) => [
+    css({
+      backgroundColor: palette.BLACK,
+    }),
+    mediaPortraitStyles,
+  ],
+
+  titlePortrait: ({ fontSizes, fontVariants, mq, spacings }: Theme) => [
+    css({
```

Comment:
Please can we apply a colour here? (The text is red in my browser)

Reply:
I've added a colour for this now, so should be fixed

Reply:
I meant a normal colour with a hex value 😄

Reply:
I'm a bit confused, what colour should this be?

Reply:
The black colour we usually use, it's currenlty inheriting the colour.

Reply:
Should it not be the colour chosen by user preference, hence why i've set it to `color: 'canvasText'`?

Reply:
CanvasText shouldn't be needed, normal colours should be able to be overridden by users perferences in this case. I am viewing without forced colours in Firefox and the strong element appears red, this is due to having set red as the text colour colour in the Firefox setting though when I then select 'Never' to not show my selected colour I still see red, this is because this element is inheriting a colour and doens't have a colour set specificaly for it in the CSS, if we add one this will fix this issue.